packages/mobile/src/components/text/text.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC, ReactElement } from 'react';
+import { Text as NativeText, TextProps as RNTextProps } from 'react-native';
+import { FontFamily, FontSize } from '~/common/enums/enums';
+
+type TextProps = RNTextProps & {
+  fontVariant?: FontFamily;
+  size?: FontSize;
+  children: string;
+};
+
+const Text: FC<TextProps> = ({
+  fontVariant = FontFamily.RALEWAY_REGULAR,
+  size = 16,
```

Comment:
I don't think we need these props. There is always an ability to set them via style, so let's keep it simple. Just set default value of Raleway font family and that's it